src/saveload/ser.rs

```diff
@@ -0,0 +1,166 @@
+use std::fmt::Display;
+use std::marker::PhantomData;
+
+use serde::ser::{self, Serialize, SerializeSeq, Serializer};
+
+use {Entities, FetchMut, Join, ReadStorage, WriteStorage};
+
+use saveload::{Components, EntityData, Storages};
+use saveload::marker::{Marker, MarkerAllocator};
+
+/// Serialize components from specified storages via `SerializableComponent::save`
+/// of all marked entities with provided serializer.
+/// When the component gets serialized with `SerializableComponent::save` method
+/// closure passed in `ids` argument mark unmarked `Entity` and they get serialized recursively.
```

Comment:
s/closure/the closure/
s/argument mark/argmuent marks/
s/\`Entity\`/Entities/?
s/they get/they will get/